### PR TITLE
Ignore claim migration for inactive tenant

### PR DESCRIPTION
**Pull request by nilasini**
Resolves:https://github.com/wso2/product-is/issues/2260
Ignoring the migration for inactive tenant, add all the local claims which have dependency with other claim dialects in migration.

### CODE REVIEW

File: modules/migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v540/migrator/ClaimDataMigrator.java [hunk 1]
```diff
@@ -94,7 +94,14 @@ public void migrate() throws MigrationClientException {
 
             // Migrate other tenants.
             List<Tenant> tenants = Utility.getTenants();
+            List<Integer> inactiveTenants = Utility.getInactiveTenants();
+            boolean ignoreForInactiveTenants = isIgnoreForInactiveTenants();
             for (Tenant tenant : tenants) {
+                int tenantId = tenant.getId();
```
Comment (darshanasbg) [inline modules/migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v540/migrator/ClaimDataMigrator.java]: Should we have this fix only for ClaimDataMigrator module? Aren't there any other modules needed this fix?

Comment (nilasini) [reply modules/migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v540/migrator/ClaimDataMigrator.java]: Currently other module migration are not depend on the tenants. We can change the migration of other modules as an impovement.

**Comment by omindu** [comment]
This will only ignore claim migration for inactive tenants, correct ?

**Comment by nilasini** [comment]
Yes it will ignore the inactive tenant according to the configuration in migration_config.yaml file.

**Pull request merged**